### DISCUSSION TITLE
Export pattern as wav file?

### DISCUSSION THREAD

**Discussion by grampalak** [hydrogen-music/hydrogen#1035] [Hydrogen on Windows]
Hello.  Nice to be here.  Seems like a new forum, or?  
I'm really enjoying using Hydrogen 1.0.1 to create home studio backing tracks using wav drum samples from my library.  I'm exporting wav song files into Audacity 2.3.2 to use Audacity's effects and tools, etc and, so far, everything's going great...BUT.  If exporting a _pattern_ in wav format isn't supported (I've tried everything I could think of), I'd like to suggest it as a feature.  I know nothing about coding but it seems as though this feature would be a snap to implement considering that hydrogen can export the full song in wav.  Or, am I missing, somehow, the existing means to do this via the toolbars or menus?
My operating system is Windows 8.1 pro.

**Comment by mauser** [comment]
Hi @grampalak,

welcome!! Yes, it was thought to be kind of a replacement for the old forum. Having it provided by GitHub solves a lot of problems for us (like hosting and spam filtering...). It is not as full-featured as the old forum, but lets see how this turns out :smiley: 

You're correct about your assumption that it is currently not possible to export one single pattern as a wav file. Well, it would be not exactly a snap, but i guess all the basics are there and it seems to be me that this feature would be nice addition to Hydrogen. Maybe you could open an issue in our issue tracker, so we can track that feature request. Otherwise things tend to go lost or forgotten so fast..

**Comment by grampalak** [comment]
Greetings Sebastian!  Well, that went better than I expected. haha!! Thanks for your positive response.  With that kind of encouragement, I believe I should just jump right in.  I'm completely new to these forums and I have no experience as far as opening an issue.  I do believe that this might be a useful feature (I could sure make use of it) and I was a little surprised when I couldn't find a way to do it.  Soooo....is there a page you can direct me to that might have a bit of instruction as to how to open an issue?  If not, I would appreciate your help.  Thanks, Jim.

**Reply by mauser** [reply]
Hi! 

I'm not sure if GitHub has a description for that, but you only need to go to "https://github.com/hydrogen-music/hydrogen/issues" and click on the green "new issue" button in the upper right corner.

**Comment by grampalak** [comment]
Done.  Thanks for your help.  :-)

**Reply by YungPhil** [reply]
Hi - I am not a techie but if I understand your query I have tried some workarounds with success in Windows 7. . 
If you want to export a single pattern, create a song using that single pattern and export to a wav. 
If you create a pattern or patterns with say latin percussion and drums or bass and want to export just the drums as a mixed track, mute the sounds you don't want in the wav and export. (conversely mute the sounds you have just exported and unmute the the remainder and repeat the process for the second track and so on.
 I do this to export drums to one wav and bass to another for import as two stereo tracks into Audacity.  If I think of a roll or break i want to add to Audacity i program the pattern, create a song from the single pattern and export.
In short you can export any sounds you want as one track byt this method. 
Hope this makes sense.
